build/Targets/VSL.Imports.targets

```diff
@@ -344,6 +344,7 @@
            or '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.12.0'
            or '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.12.1.DesignTime'
            or '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.14.0.DesignTime'
+           or '%(FileName)' == 'Microsoft.VisualStudio.Shell.Interop.15.0.DesignTime'
```

Comment:
As a side note, I feel like the SDK providing a general mechanism for this would be useful